template_feed/Microsoft.DotNet.Common.ItemTemplates/content/RazorPageWithPageModel/Index.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
```

Comment:
For templates, we generally only include the usings that are required by the generated template code. But I could see including additional usings that most users would end up adding as they modify the content.